src/js/components/TaskMesosUrlComponent.jsx

```diff
@@ -64,7 +64,9 @@ var TaskMesosUrlComponent = React.createClass({
     var text = this.props.text;
     var mesosTaskUrl = this.buildUrl(task, info);
     if (mesosTaskUrl == null) {
-      return null;
+      return (
+        <span className="text-muted">No link available</span>
```

Comment:
Yes, slightly, if there is now content in the dd-element, there is no line break and the design blows up.